src/contracts/pods/EigenPodManager.sol

```diff
@@ -29,14 +31,27 @@ import "./EigenPodPausingConstants.sol";
  * - keeping track of the balances of all validators of EigenPods, and their stake in EigenLayer
  * - withdrawing eth when withdrawals are initiated
  */
-contract EigenPodManager is Initializable, OwnableUpgradeable, Pausable, IEigenPodManager, EigenPodPausingConstants {
+contract EigenPodManager is 
+    Initializable, 
+    OwnableUpgradeable, 
+    Pausable, 
+    IEigenPodManager,
+    EigenPodPausingConstants,
+    ReentrancyGuardUpgradeable
```

Comment:
I'm fairly certain that adding this into the inheritance graph of the EigenPodManager will destroy its storage pattern by shifting it downward 50 slots.
might need to do something like moving the storage into a separate contract and inheriting from `ReentrancyGuardUpgradeable` further down the tree?
please discuss with me if you are planning on doing this / unsure of the best approach here

Reply:
From @ChaoticWalrus :
I think:
move the EigenPodManager storage (other than immutables) into a simple, standalone contract (EigenPodManagerStorage , don’t have it inherit from anything)
make the inheritance graph look like:
    Initializable, 
    OwnableUpgradeable, 
    Pausable, 
    IEigenPodManager,
    EigenPodPausingConstants,
    EigenPodManagerStorage,
    ReentrancyGuardUpgradeable

Reply:
this works because all the storage is inherited before the ReentrancyGuardUpgradeable

Reply:
fixed!